contexts/MembershipContext/tests/Data/ValidMembershipApplication.php

```diff
@@ -171,7 +178,7 @@ private function newPayPalPayment() {
 		return new Payment(
 			self::PAYMENT_PERIOD_IN_MONTHS,
 			Euro::newFromFloat( self::PAYMENT_AMOUNT_IN_EURO ),
-			new PayPalPayment()
+			new PayPalPayment( ( new PayPalData() )->freeze() )
```

Comment:
Right, totally unnecessary. I changed it (and also the `MembershipApplicationBuilder`).